agent/structs/acl.go

```diff
@@ -1269,6 +1269,7 @@ type ACLReplicationStatus struct {
 	ReplicatedTokenIndex uint64
 	LastSuccess          time.Time
 	LastError            time.Time
+	LastErrorMessage     string
```

Comment:
There is unfortunately another copy of this struct in `api/acl.go` which will need this field added to it as well. That copy of `ACLReplicationStatus` is used by the `api` client library package to deserialize the API response.

Reply:
Done.